lib/assessments/permission.rb

```diff
@@ -73,23 +77,23 @@ def revoke_level(user)
           assessment.viewers.destroy(user)
         when :network_partner
           assessment.network_partners.destroy(user)
-        assessment.reload
+          assessment.reload
```

Comment:
Okay - quick smoke test of general Ruby switch statements indicates that we're okay here.  Conventionally you only see one line of action inside of a `when` which is what gave me pause to begin with, but I believe that we're alright.